src/user-guide/running-suites.rst

```diff
@@ -372,25 +372,33 @@ for task messaging by TCP or SSH. See :ref:`Polling To Track Job Status`.
 Tracking Task State
 -------------------
 
-Cylc supports two ways of tracking task state on job hosts:
+Cylc supports three ways of tracking task state on job hosts:
 
-- task-to-suite messaging via TCP (using ZMQ protocol)
+- task-to-workflow messaging via TCP (using ZMQ protocol)
+- task-to-workflow messaging via non-interactive SSH to the workflow host, then
+  local tcp.
 - regular polling by the :term:`scheduler`
 
-These can be configured per job host using
+These can be configured per platform using
 :cylc:conf:`global.cylc[platforms][<platform name>]communication method`.
 
 If your site prohibits TCP and SSH back from job hosts to
-suite hosts, before resorting to the polling method you should
+workflow hosts, before resorting to the polling method you should
 consider installing dedicated Cylc servers or
 VMs inside the HPC trust zone (where TCP and SSH should be allowed).
 
 It is also possible to run Cylc :term:`schedulers <scheduler>` on HPC login
-nodes, but this is not recommended for load and run duration,
+nodes, but this is not recommended for load and run duration.
```

Comment:
I'm not sure that I understand what this means. I think that it might be trying to say:
"...not recommended because the scheduler may run for longer and use more resources than expected on a login node."

Reply:
> "...not recommended because the scheduler may run for longer and use more resources than expected on a login node."

I don't think that's what was meant.